instrumentation/src/main/java/io/opentelemetry/android/OpenTelemetryRumBuilder.java

```diff
@@ -308,7 +317,32 @@ private SdkTracerProvider buildTracerProvider(SessionId sessionId, Application a
     private SpanExporter buildSpanExporter() {
         // TODO: Default to otlp...but how can we make endpoint and auth mandatory?
         SpanExporter defaultExporter = LoggingSpanExporter.create();
-        return spanExporterCustomizer.apply(defaultExporter);
+        SpanExporter spanExporter = defaultExporter;
+        DiskBufferingConfiguration diskBufferingConfiguration =
+                config.getDiskBufferingConfiguration();
+        if (diskBufferingConfiguration.isEnabled()) {
+            try {
+                spanExporter = createDiskExporter(defaultExporter, diskBufferingConfiguration);
+            } catch (IOException e) {
+                LOGGER.log(Level.WARNING, "Could not create span disk exporter.", e);
```

Comment:
Actually, this makes me wonder if this shouldn't be using `android.util.Log` like the rest of the library. I'm noticing that the only places using `jul` are the `CacheStorageService` and the `DiskManager`. Is this intentional?

Reply:
This is a good point! Tbh I didn't notice the usage of Android Log until now, so I thought we'd configure some other framework to delegate to the Android Log at some point, but it seems like it'd be better to do so with slf4j or timber rather than jul now that I checked better. In any case, I'm fine with using the Android Log class directly, I've just replaced jul usages by Android Log ones across the project 👍

Reply:
Cool, yeah I figure we'll have to pick this up eventually when we look at #142.